docs/more-pushmsg.md

```diff
@@ -33,3 +33,66 @@ func main() {
         url.Values{"message": {"My Message"}, "title": {"My Title"}})
 }
 ```
+
+### PHP (using cURL)
+
+```php
+$data = array(
+    "title"=> "Hello World",
+    "message"=> "Test push From PHP cURL.",
+    "priority"=> 5,
+);
+
+$data_string = json_encode($data);
+
+$url = "http://localhost:8008/message?token=<apptoken>";
+
+$headers = array
+(
+    'Content-Type: application/json; charset=utf-8'
+);
+
+$ch = curl_init();
+curl_setopt($ch, CURLOPT_URL, $url);
+curl_setopt($ch, CURLOPT_POST, 1);
+curl_setopt( $ch,CURLOPT_HTTPHEADER, $headers );
+curl_setopt( $ch,CURLOPT_RETURNTRANSFER, true );
+curl_setopt( $ch,CURLOPT_POSTFIELDS, $data_string);
+
+$result = curl_exec($ch);
+$code = curl_getinfo($ch, CURLINFO_HTTP_CODE);
+
+if ($code == 400)
```

Comment:
Why not use a switch … case … for this?

Reply:
Done It Validate using HTTP status code `https://gotify.net/api-docs#/message/createMessage`

```php
$code = curl_getinfo($ch, CURLINFO_HTTP_CODE);
```